### ISSUE TITLE
Use the output of --live_transcribe

### ISSUE THREAD

**Issue by fatualux** [Softcatala/whisper-ctranslate2#52]
Hi, and thanks for your project, it works flawlessly and I had met no issue at all.
I would just like to ask if there is a way to "trap" or echo somewhere the real time transcriptions obtained with the option `--live_transcribe True`
Thanks in advance for your support

**Comment by ArnaudKunzi** [comment]
Sounds like a job for [tee](https://www.gnu.org/software/coreutils/manual/html_node/tee-invocation.html).

**Comment by fatualux** [comment]
Thanks I did not know this package. I will try to write a sctipt to integrate tee with --live_transcribe.

**Comment by mpuz** [comment]
> Hi, and thanks for your project, it works flawlessly and I had met no issue at all. I would just like to ask if there is a way to "trap" or echo somewhere the real time transcriptions obtained with the option `--live_transcribe True` Thanks in advance for your support

Looks like You are the happy one who succeed to have live-transcribe to work, can  You share your cli args and what language do You transcribe?

**Comment by fatualux** [comment]
yes, of course. I transcribe Italian... Here is the link to the script I am
now using for live transcription. It is a bit slow in transcribing, so I
always use small model.

https://github.com/fatualux/scripts/blob/main/realtimeWhisper.sh

..:FZ:..

- Sent from my Android device -

Il gio 19 ott 2023, 20:44 Michael ***@***.***> ha scritto:

**Comment by P3NG00** [comment]
> > Hi, and thanks for your project, it works flawlessly and I had met no issue at all. I would just like to ask if there is a way to "trap" or echo somewhere the real time transcriptions obtained with the option `--live_transcribe True` Thanks in advance for your support
> 
> Looks like You are the happy one who succeed to have live-transcribe to work, can You share your cli args and what language do You transcribe?
> 
> for saving output You can feed it to the output to file - read help
> 
> ```
> whisper-ctranslate2 --help
> ```
> 
> take a look at
> 
> ```
> Configuration options to control generated outputs:
>   --output_dir OUTPUT_DIR, -o OUTPUT_DIR
>                         directory to save the outputs (default: .)
>   --output_format {txt,vtt,srt,tsv,json,all}, -f {txt,vtt,srt,tsv,json,all}
>                         format of the output file; if not specified, all
>                         available formats will be produced (default: all)
>   --pretty_json PRETTY_JSON, -p PRETTY_JSON
>                         produce json in a human redable format (default:
>                         False)
> ```

I cannot get the output format or output dir to work for --live_transcribe. Tee barely works, I would like some way to just get the text output so that I can put it into another program.
